.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "PlanthorIdentityServer-ws",
+    "dockerComposeFile": "docker-compose.yml",
+    "service": "planthoridentity",
+    "workspaceFolder": "/workspace/PlanthorIdentityServer",
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-dotnettools.csharp",
+				"ms-dotnettools.csdevkit"
+			]
+		}
+	},
+	//Expose port here
+	"forwardPorts": [5000,5001,5432,8888],
+	"postCreateCommand": ".devcontainer/scripts/postCreateCommand.sh",
+	"remoteUser": "vscode"
+	
+	// [Optional] To reuse of your local HTTPS dev cert:
```

Comment:
Only comment the important idea, don't overflow file with a lot of comment, hard to maintain